.circleci/config.yml

```diff
@@ -558,31 +558,31 @@ workflows:
             branches:
               ignore: /.*/
             tags:
-              only: /^v[0-9]+\.[0-9]+\.[0-9]+-(rc|alpha|beta)\.[0-9]+$/
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+(-(rc|alpha|beta)\.[0-9]+)?/
```

Comment:
```suggestion
              only: /^v[0-9]+\.[0-9]+\.[0-9]+(-(rc|alpha|beta)\.[0-9]+)?$/
```

Reply:
@russorat you are suggesting to keep a `$` at the end?

Reply:
Added `$` sign at the end of the expression